### DISCUSSION TITLE
Livewire load slowly in prod

### DISCUSSION THREAD

**Discussion by clementlanot** [livewire/livewire#7238] [1. Help]
Hi, small problem on my site. Livewire works perfectly well locally. But in production at OVH, the script does not seem to load. It takes more than 5s and timeout.

I can directly access the script by going to the page https://xxx.fr/livewire/livewire.js and it is very slow to load but the script is accessible.

I tried :

` php artisan vendor:publish --force --tag=livewire:assets`
And :

```
"scripts": {
"post-autoload-dump": [
	"Illuminate\Foundation\ComposerScripts::postAutoloadDump",
	"@php artisan package:discover --ansi",
	"@php artisan vendor:publish --force --tag=livewire:assets --ansi"
],
```

But it does not work. 

I wonder if my shared server is no longer powerful enough or if there is a limit from my shared hosting ?

Any help ? thx!

**Comment by alucard29** [comment]
Hi, Same issue here. Did you find a solution ?

**Comment by clementlanot** [comment]
No, I left my host OVH for Infomaniak and everything is working fine.

**Comment by alucard29** [comment]
Ok thx !
